### PR TITLE
Adding macOS impeller info

### PR DESCRIPTION
**Pull request by leighajarett**
Note that this should go live for 3.13

_Issues fixed by this PR (if any):_ https://github.com/flutter/website/issues/8908

## Presubmit checklist

- [x] This PR doesn’t contain automatically generated corrections (Grammarly or similar).
- [x] This PR follows the [Google Developer Documentation Style Guidelines](https://developers.google.com/style) — for example, it doesn’t use _i.e._ or _e.g._, and it avoids _I_ and _we_ (first person).
- [x] This PR uses [semantic line breaks](https://github.com/dart-lang/site-shared/blob/main/doc/writing-for-dart-and-flutter-websites.md#semantic-line-breaks) of 80 characters or fewer.

### CODE REVIEW

**Comment by leighajarett** [comment]
@chinmaygarde FYI. @Sfshaza this is for the next stable release

File: src/perf/impeller.md [hunk 1]
```diff
@@ -74,6 +74,44 @@ include a small reproducible test case.
 
 [file-issue]: {{site.github}}/flutter/flutter/issues/new/choose
 
+### macOS
```
Comment (sfshaza2) [inline src/perf/impeller.md:77]: Does Impeller work on iOS mobile as well as macOS desktop? I see both used here.

Comment (chinmaygarde) [reply src/perf/impeller.md:77]: Yes. We are moving to preview Impeller on macOS desktop.

File: src/perf/impeller.md [hunk 2]
```diff
@@ -74,6 +74,44 @@ include a small reproducible test case.
...
+
+Impeller is available for macOS in preview as of the 
+Flutter 3.13 stable release.
+
+* To _enable_ Impeller on iOS when debugging,
```
Comment (chinmaygarde) [inline src/perf/impeller.md:82]: Here and elsewhere, there are a couple of instances where the patch refers to iOS :)

Comment (leighajarett) [reply src/perf/impeller.md]: Whoops thanks!

File: src/perf/impeller.md (hunk 1 above)
Comment (leighajarett) [reply src/perf/impeller.md:77]: Yeah we already have iOS support, macOS preview is being added in 3.13

**Comment by chinmaygarde** [comment]
lgtm will defer to lgt @sfshaza2

File: src/perf/impeller.md [hunk 3]
```diff
@@ -74,6 +74,44 @@ include a small reproducible test case.
...
+with Impeller on macOS, file an issue in the [GitHub tracker][file-issue].
+Prefix the issue title with `[Impeller]` and
+include a small reproducible test case.
+
+It's helpful to include the following information when 
+submitting an issue for Impeller on macOS:
+
+- The device you are running on, including the chip information.
+- Screenshots or recordings of any visible issues.
+- An [export of the performance trace][]. Please add it to the Github issue by zipping the file.
```
Comment (sfshaza2) [inline src/perf/impeller.md:109]:
```suggestion
- An [export of the performance trace][]. Please add it to the GitHub issue by zipping the file.
```

File: src/perf/impeller.md [hunk 4]
```diff
@@ -74,6 +74,44 @@ include a small reproducible test case.
...
+    <true />
+  ```
+
+The team continues to improve macOS support.
+If you encounter performance or fidelity issues
+with Impeller on macOS, file an issue in the [GitHub tracker][file-issue].
+Prefix the issue title with `[Impeller]` and
+include a small reproducible test case.
+
+It's helpful to include the following information when
```
Comment (sfshaza2) [inline src/perf/impeller.md:104]:
```suggestion
Please include the following information when 
```

File: src/perf/impeller.md (hunk 1 above)
Comment (sfshaza2) [reply src/perf/impeller.md:77]: Great!

**Review by sfshaza2 — APPROVED** [review]
lgtm with a few minor fixes

**Pull request merged**